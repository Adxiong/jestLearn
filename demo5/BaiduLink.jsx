/*
 * @Description: 
 * @version: 
 * @Author: Adxiong
 * @Date: 2022-05-25 22:13:19
 * @LastEditors: Adxiong
 * @LastEditTime: 2022-05-25 22:38:09
 */
import React from 'react'


const BaiduLink = () => {
  const title = "百度一下,你就知道了"
  const url = 'https://www.baidu.com'
  return (
    <div>
      <a href={url}>{title}</a>
    </div>
  )
}

export default BaiduLink