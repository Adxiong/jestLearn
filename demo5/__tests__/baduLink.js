/*
 * @Description: 
 * @version: 
 * @Author: Adxiong
 * @Date: 2022-05-25 22:21:11
 * @LastEditors: Adxiong
 * @LastEditTime: 2022-05-25 22:35:48
 */
import BaiduLink from "../BaiduLink";
import renderer from "react-test-renderer";
import React from 'react'

test("Baidulink component render content test", () => {
  const tree = renderer.create(<BaiduLink></BaiduLink>).toJSON()
  expect(tree).toMatchSnapshot()
})