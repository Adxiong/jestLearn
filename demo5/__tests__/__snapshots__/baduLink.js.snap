// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Baidulink component render content test 1`] = `
<div>
  <a
    href="https://www.baidu.com"
  >
    百度一下,你就知道了
  </a>
</div>
`;
